docs/widgets.md

```diff
@@ -57,6 +57,7 @@ Dashy has support for displaying dynamic content in the form of widgets. There a
   - [Nextcloud PHP OPcache](#nextcloud-php-opcache-stats)
   - [Sabnzbd](#sabnzbd)
   - [Gluetun VPN Info](#gluetun-vpn-info)
+  - [Drone.io](#drone-io-builds)
```

Comment:
In case it's helpful, you can test how it renders by previewing it in the GH UI, 
Currently like: https://github.com/Lissy93/dashy/blob/f0a5c249b0261acf594ca0379e60836ca0b0746a/docs/widgets.md#droneio-builds